### ISSUE TITLE
How to catch errors on Login?

### ISSUE THREAD

**Issue by 0xs34n** [tiberiuc/redux-react-firebase#32]
firebase.login() does not seem to change the redux state of `authError`. How do I know if the user has inputted an invalid username / password?

**Comment by 0xs34n** [comment]
By the way, I am trying to follow the provided example

``` javascript

@firebase()
@connect(
  ({firebase}) => ({
    authError: pathToJS(firebase, 'authError'),
  })
)
class Login extends Component {
  constructor(){
    super()
    this.state = {loading:false}
  }

  render(){
    const {firebase, authError} = this.props

    const handleLogin = () => {
      const {email, password} = this.refs

      const credentials = {
        email: email.value,
        password: password.value
      }


      firebase.login(credentials)
      this.state.loading = true
      this.setState(this.state)
    }

    const error = (authError) ?
                      authError.toString()
                    : (this.state.loading) ?
                              'loading'
                            : ''

    return(
      <div>
        <h1>Login</h1>
        <input type='email' placeholder='Email' ref='email' /><br/>
        <input type='password' placeholder='Password' ref='password' />
        <p>{error}</p>
        <button onClick={handleLogin}>Login</button>
      </div>
    )
  }
}
```

It seems like authError acts as a listener, but when I try it myself, `authError` is always null

**Comment by tiberiuc** [comment]
authError should be set only when there is a problem on login ( like user and or password are incorrect ).

I will look myself to see if there is a bug

**Comment by davestevens** [comment]
This should be resolved in 2.3.1 https://github.com/tiberiuc/redux-react-firebase/releases/tag/2.3.1
